### PR TITLE
#1 - ClientSlices and Settings interfaces

### PR DESCRIPTION
**Pull request by olegmoz**
Part of #1 
Added basic `ClientSlices` and `Settings` interfaces

### CODE REVIEW

File: src/main/java/com/artipie/http/client/ClientSlices.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+/**
+ * Slices collection that provides client slices by host and port.
+ *
+ * @since 0.1
+ */
+public interface ClientSlices {
+
+    /**
+     * Create client slice sending HTTP requests to host specified on port 80.
```
Comment (olenagerasimova) [inline src/main/java/com/artipie/http/client/ClientSlices.java:36]: @olegmoz hm, sentence seems off,  I'd suggest to place word "specified" before "host"

File: src/main/java/com/artipie/http/client/ClientSlices.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+     *
+     * @param host Host name.
+     * @return Client slice.
+     */
+    Slice http(String host);
+
+    /**
+     * Create client slice sending HTTP requests to host specified.
```
Comment (olenagerasimova) [inline src/main/java/com/artipie/http/client/ClientSlices.java:44]: @olegmoz this sentence seems unfinished. Also, please check other javadocs, seems like some corrections are required.

**Comment by olegmoz** [comment]
@olenagerasimova thanks, fixed the comments

**Comment by olegmoz** [comment]
@g4s8 please see this PR

**Comment by g4s8** [comment]
Now @artipie/contributors can approve pull requests.

**Pull request merged**